src/common/blkdev.cc

```diff
@@ -400,8 +400,12 @@ int BlkDev::block_device_model(char *model, size_t max)
   // and "ATA device_model" for ATA disks.  Some vendors choose to put the
   // vendor name in device_model, and some don't.  Strip the first bit.
   p = arg.value.str;
-  (void) strsep(&p, " ");
-  snprintf(model, max, "%s", p);
+  if (p == NULL || *p == '\0') {
```

Comment:
I think `p == NULL` is not possible  here, but it is possible below, after `strsep`, if the initial string does not contain " ". So I think you need to check after `strsep`.